visualization/mayavi_vis_utils.py

```diff
@@ -0,0 +1,117 @@
+
+"""
+Utilities for rendering camera frustums and 3d point clouds using Mayavi mlab.
+
+Author: John Lambert
+"""
+
+import argparse
+from typing import List
+
+import mayavi
+import numpy as np
+from colour import Color
+from gtsam import Cal3Bundler, Pose3
+from mayavi import mlab
+
+
+from gtsfm.common.view_frustum import ViewFrustum
+
+
+def draw_point_cloud_mayavi(
+    sphere_radius: float, fig: mayavi.core.scene.Scene, point_cloud: np.ndarray, rgb: np.ndarray
+) -> None:
+    """Render a point cloud as a collection of spheres, using Mayavi.
+
+    Args:
+        sphere_radius: radius of each rendered sphere.
+        fig: Mayavi figure object.
+        point_cloud: array of shape (N,3) representing 3d points.
+        rgb: uint8 array of shape (N,3) representing colors in RGB order, in the range [0,255]
```

Comment:
I think point_cloud and rgb should come before sphere_radius in args list.

Reply:
good point, updated